### DISCUSSION TITLE
Chat bot

### DISCUSSION THREAD

**Discussion by mak7ilenin** [VirumaaAnts/-Morality#5] [Q&A] [answered]
Are we making the chat bot like a vk bot or it will be just an answer on question?

**Comment by mak7ilenin** [comment]
I think that would be better if we will do the chat like a vk bot with an opportunity to choose..

**Comment by 9I6JloKo** [comment, marked as answer]
I think that would be better if we will do the chat like a vk bot with an opportunity to choose..
